lib/anagrams.rb

```diff
@@ -0,0 +1,38 @@
+class Anagrams
+  def initialize(anagram_file)
```

Comment:
@zzokki81 `anagram_file` is no longer a good name for this var. At the least it should be something more intention revealing like `dictionary` or `words`, you can even go overboard (it's just a local var/argument) and call it `words_array_or_file`.